### DISCUSSION TITLE
希望.md文件支持侧边目录树功能

### DISCUSSION THREAD

**Discussion by Pandolar** [AlistGo/alist#249] [Ideas]
请问有没有支持markdown在线预览时候侧边目录树功能的想法呢
例:
![image](https://user-images.githubusercontent.com/54896726/147460059-e9e56c5b-7c48-4e9f-915c-386fa48ea5c3.png)

**Comment by xhofe** [comment]
暂无此计划，毕竟这是一个目录列表程序，并不是主要markdown查看的。
会优先写其他的计划的功能，https://github.com/Xhofe/alist/projects/1 ，其他的完成之后可能会添加。
